package.json

```diff
@@ -10,7 +10,9 @@
     "lint": "./node_modules/.bin/eslint src",
     "test:watch": "./node_modules/.bin/jest --watch --coverage=false",
     "precommit": "./node_modules/.bin/lint-staged",
-    "machine-gun": "node tools/machine-gun.js"
+    "machine-gun": "node tools/machine-gun.js",
```

Comment:
outdated, no? this is [here](https://github.com/leapdao/leap-guardian/blob/master/scripts/machineGun.js) now.

Reply:
Right 😺. I also removed the non-functional and outdated `nst-transfer` 👍